  INFO root [2018-07-02 15:14:23,928] - 
  INFO root [2018-07-02 15:14:23,930] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 15:14:26,774] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 15:14:26,775] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 15:14:26,775] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 15:14:27,161] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 15:14:27,353] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 15:14:27,622] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 15410. PID: 2860 ----
  INFO root [2018-07-02 15:14:27,622] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 15:14:27,668] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 15:14:27,669] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 15:14:33,902] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 15:14:34,111] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 15:14:34,256] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 15:14:35,320] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 15:14:35,403] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 15:14:35,438] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 15:14:35,661] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 15:14:37,927] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 15:14:40,773] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 15:14:40,774] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 15:14:41,513] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 15:14:41,878] - TelnetServer server is running on port 33000
  WARN root [2018-07-02 15:14:41,940] - c:\users\public\documents\workspace\kbe\kbengine-kbengine-v1.0.0\kbengine\kbe\src\server\tools\interfaces\interfaces.cpp:164: Profile scriptCall took 0.37 seconds
  INFO root [2018-07-02 15:14:42,697] - ---- interfaces is running ----
 DEBUG root [2018-07-02 15:14:42,719] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 15:14:50,057] - Components::process(): Found all the components!
S_INFO root [2018-07-02 15:15:37,931] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 15:42:09,248] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 15:46:30,453] - onRequestAccountLogin: registerName=lqt02
  INFO root [2018-07-02 16:30:54,061] - 
  INFO root [2018-07-02 16:30:54,063] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 16:30:54,064] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 16:30:54,065] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 16:30:54,066] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 16:30:54,102] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 16:30:54,102] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 16:30:54,145] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 30435. PID: 3984 ----
  INFO root [2018-07-02 16:30:54,147] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 16:30:54,147] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 16:30:54,148] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 16:30:54,365] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 16:30:54,366] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 16:30:54,367] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 16:30:54,374] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 16:30:54,375] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 16:30:54,376] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 16:30:54,377] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 16:30:54,529] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 16:30:54,686] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 16:30:54,686] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 16:30:54,737] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
 S_ERR root [2018-07-02 16:30:54,807] - Traceback (most recent call last):
  File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\kbemain.py", line 45, in onInterfaceAppReady
    g_poller.start("localhost", 30040)
 S_ERR root [2018-07-02 16:30:54,819] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\Poller.py", line 30, in start
    self._socket.bind((addr, port))
 S_ERR root [2018-07-02 16:30:54,820] - OSError: [WinError 10048] 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
  INFO root [2018-07-02 16:30:54,827] - TelnetServer server is running on port 33001
  INFO root [2018-07-02 16:30:55,303] - ---- interfaces is running ----
 DEBUG root [2018-07-02 16:30:55,304] - Components::process(): Request for the process of identity...
 ERROR root [2018-07-02 16:30:55,428] - Components::process: found interfaces, addr:192.168.31.56:30099
 ERROR root [2018-07-02 16:30:55,475] - ServerApp::onIdentityillegal: The current process and interfaces(componentID=3000 ->conflicted???, pid=2860, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [2018-07-02 16:30:55,476] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [2018-07-02 16:30:55,476] - onInterfaceAppShutDown()
  INFO root [2018-07-02 16:30:56,601] - Shutdowner::onShutdown: shutting down
  WARN root [2018-07-02 16:30:56,706] - c:\users\public\documents\workspace\kbe\kbengine-kbengine-v1.0.0\kbengine\kbe\src\lib\network\event_dispatcher.cpp:171: Profile callSystemTimers took 0.23 seconds
  INFO root [2018-07-02 16:30:57,707] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-07-02 16:30:57,742] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 16:30:57,742] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 16:30:57,743] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 16:30:57,743] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 16:30:57,743] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 16:30:57,743] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 16:30:57,744] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 16:30:57,744] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 16:30:57,772] - Script::uninstall(): is successfully!
 DEBUG root [2018-07-02 16:30:57,783] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-07-02 16:30:58,083] - ThreadPool::destroy(): waiting for thread(2)[0x921410,0x9213a8,], try=1
 DEBUG root [2018-07-02 16:30:58,483] - TPThread::~TPThread(): 0x921410
 DEBUG root [2018-07-02 16:30:58,483] - TPThread::~TPThread(): 0x9213a8
 DEBUG root [2018-07-02 16:30:58,483] - ThreadPool::destroy(): successfully!
 DEBUG root [2018-07-02 16:30:58,628] - Bundle::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:30:58,629] - Channel::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:30:58,629] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:30:58,629] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2018-07-02 16:30:58,629] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:30:58,666] - Address::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:30:58,667] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:30:58,667] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2018-07-02 16:30:58,667] - interfaces(3000) has shut down.
  INFO root [2018-07-02 16:30:59,667] - EventDispatcher()::~EventDispatcher: Num timers = 2
  WARN root [2018-07-02 16:31:01,792] - TCPPacketReceiver::processPendingEvents(127.0.0.1:12256): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 16:31:07,339] - 
  INFO root [2018-07-02 16:31:07,350] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 16:31:07,352] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 16:31:07,353] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 16:31:07,353] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 16:31:07,417] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 16:31:07,417] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 16:31:07,428] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 30481. PID: 6624 ----
  INFO root [2018-07-02 16:31:07,428] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 16:31:07,433] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 16:31:07,435] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 16:31:07,612] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 16:31:07,613] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 16:31:07,614] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 16:31:07,619] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 16:31:07,620] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 16:31:07,620] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 16:31:07,622] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 16:31:07,688] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 16:31:07,757] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 16:31:07,758] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 16:31:07,777] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
 S_ERR root [2018-07-02 16:31:07,791] - Traceback (most recent call last):
  File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\kbemain.py", line 45, in onInterfaceAppReady
    g_poller.start("localhost", 30040)
 S_ERR root [2018-07-02 16:31:07,793] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\Poller.py", line 30, in start
    self._socket.bind((addr, port))
 S_ERR root [2018-07-02 16:31:07,795] - OSError: [WinError 10048] 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
  INFO root [2018-07-02 16:31:07,799] - TelnetServer server is running on port 33001
  INFO root [2018-07-02 16:31:08,355] - ---- interfaces is running ----
 DEBUG root [2018-07-02 16:31:08,356] - Components::process(): Request for the process of identity...
 ERROR root [2018-07-02 16:31:08,492] - Components::process: found interfaces, addr:192.168.31.56:30099
 ERROR root [2018-07-02 16:31:08,494] - ServerApp::onIdentityillegal: The current process and interfaces(componentID=3000 ->conflicted???, pid=2860, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [2018-07-02 16:31:08,496] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [2018-07-02 16:31:08,496] - onInterfaceAppShutDown()
  INFO root [2018-07-02 16:31:09,497] - Shutdowner::onShutdown: shutting down
  INFO root [2018-07-02 16:31:10,499] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-07-02 16:31:10,500] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 16:31:10,501] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 16:31:10,501] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 16:31:10,501] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 16:31:10,501] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 16:31:10,502] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 16:31:10,502] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 16:31:10,502] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 16:31:10,523] - Script::uninstall(): is successfully!
 DEBUG root [2018-07-02 16:31:10,523] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-07-02 16:31:10,823] - ThreadPool::destroy(): waiting for thread(2)[0x5a11f0,0x5a1188,], try=1
 DEBUG root [2018-07-02 16:31:11,224] - TPThread::~TPThread(): 0x5a11f0
 DEBUG root [2018-07-02 16:31:11,225] - TPThread::~TPThread(): 0x5a1188
 DEBUG root [2018-07-02 16:31:11,225] - ThreadPool::destroy(): successfully!
 DEBUG root [2018-07-02 16:31:11,253] - Bundle::destroyObjPool(): size 1.
 DEBUG root [2018-07-02 16:31:11,253] - Channel::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:31:11,253] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:31:11,254] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2018-07-02 16:31:11,254] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:31:11,254] - Address::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:31:11,255] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:31:11,255] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2018-07-02 16:31:11,255] - interfaces(3000) has shut down.
  INFO root [2018-07-02 16:31:12,255] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2018-07-02 16:32:44,100] - 
  INFO root [2018-07-02 16:32:44,101] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 16:32:44,103] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 16:32:44,104] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 16:32:44,104] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 16:32:44,155] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 16:32:44,155] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 16:32:44,187] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 30794. PID: 9492 ----
  INFO root [2018-07-02 16:32:44,190] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 16:32:44,191] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 16:32:44,191] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 16:32:44,372] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 16:32:44,373] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 16:32:44,374] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 16:32:44,381] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 16:32:44,382] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 16:32:44,382] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 16:32:44,384] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 16:32:44,465] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 16:32:44,547] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 16:32:44,548] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 16:32:44,574] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
 S_ERR root [2018-07-02 16:32:44,601] - Traceback (most recent call last):
  File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\kbemain.py", line 45, in onInterfaceAppReady
    g_poller.start("localhost", 30040)
 S_ERR root [2018-07-02 16:32:44,603] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\Poller.py", line 30, in start
    self._socket.bind((addr, port))
 S_ERR root [2018-07-02 16:32:44,605] - OSError: [WinError 10048] 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
  INFO root [2018-07-02 16:32:44,610] - TelnetServer server is running on port 33001
  INFO root [2018-07-02 16:32:45,161] - ---- interfaces is running ----
 DEBUG root [2018-07-02 16:32:45,161] - Components::process(): Request for the process of identity...
 ERROR root [2018-07-02 16:32:45,260] - Components::process: found interfaces, addr:192.168.31.56:30099
 ERROR root [2018-07-02 16:32:45,261] - ServerApp::onIdentityillegal: The current process and interfaces(componentID=3000 ->conflicted???, pid=2860, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [2018-07-02 16:32:45,263] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [2018-07-02 16:32:45,263] - onInterfaceAppShutDown()
  INFO root [2018-07-02 16:32:46,264] - Shutdowner::onShutdown: shutting down
  INFO root [2018-07-02 16:32:47,265] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-07-02 16:32:47,266] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 16:32:47,266] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 16:32:47,267] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 16:32:47,267] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 16:32:47,267] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 16:32:47,267] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 16:32:47,268] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 16:32:47,268] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 16:32:47,286] - Script::uninstall(): is successfully!
 DEBUG root [2018-07-02 16:32:47,287] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-07-02 16:32:47,587] - ThreadPool::destroy(): waiting for thread(2)[0x1e11f0,0x1e1188,], try=1
 DEBUG root [2018-07-02 16:32:47,987] - TPThread::~TPThread(): 0x1e11f0
 DEBUG root [2018-07-02 16:32:47,987] - TPThread::~TPThread(): 0x1e1188
 DEBUG root [2018-07-02 16:32:47,987] - ThreadPool::destroy(): successfully!
 DEBUG root [2018-07-02 16:32:48,019] - Bundle::destroyObjPool(): size 1.
 DEBUG root [2018-07-02 16:32:48,020] - Channel::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:32:48,021] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:32:48,021] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2018-07-02 16:32:48,021] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:32:48,021] - Address::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:32:48,022] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 16:32:48,022] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2018-07-02 16:32:48,022] - interfaces(3000) has shut down.
  INFO root [2018-07-02 16:32:49,022] - EventDispatcher()::~EventDispatcher: Num timers = 2
  WARN root [2018-07-02 16:34:40,527] - TCPPacketReceiver::processPendingEvents(127.0.0.1:11005): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 18:03:14,722] - 
  INFO root [2018-07-02 18:03:14,722] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 18:03:14,738] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 18:03:14,738] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 18:03:14,738] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 18:03:14,753] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 18:03:14,753] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 18:03:14,753] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 15762. PID: 4680 ----
  INFO root [2018-07-02 18:03:14,753] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 18:03:14,753] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 18:03:14,753] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 18:03:14,940] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 18:03:14,940] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 18:03:14,940] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 18:03:14,956] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 18:03:14,956] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 18:03:14,956] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 18:03:14,956] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 18:03:15,050] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 18:03:15,128] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 18:03:15,128] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 18:03:15,159] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 18:03:15,174] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 18:03:16,048] - ---- interfaces is running ----
 DEBUG root [2018-07-02 18:03:16,048] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 18:03:23,365] - Components::process(): Found all the components!
S_INFO root [2018-07-02 18:14:39,585] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 18:15:04,586] - onRequestAccountLogin: registerName=lqt02
  WARN root [2018-07-02 18:27:30,982] - TCPPacketReceiver::processPendingEvents(127.0.0.1:1346): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 18:27:36,801] - 
  INFO root [2018-07-02 18:27:36,814] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 18:27:36,816] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 18:27:36,817] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 18:27:36,817] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 18:27:36,859] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 18:27:36,860] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 18:27:36,871] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 20536. PID: 6928 ----
  INFO root [2018-07-02 18:27:36,873] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 18:27:36,877] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 18:27:36,877] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 18:27:37,073] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 18:27:37,075] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 18:27:37,076] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 18:27:37,084] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 18:27:37,085] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 18:27:37,085] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 18:27:37,087] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 18:27:37,176] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 18:27:37,266] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 18:27:37,267] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 18:27:37,293] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 18:27:37,315] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 18:27:38,272] - ---- interfaces is running ----
 DEBUG root [2018-07-02 18:27:38,273] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 18:27:45,494] - Components::process(): Found all the components!
S_INFO root [2018-07-02 18:28:15,395] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 18:28:28,895] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 18:37:27,914] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 18:37:41,014] - onRequestAccountLogin: registerName=lqt01
  WARN root [2018-07-02 18:39:17,262] - TCPPacketReceiver::processPendingEvents(127.0.0.1:1978): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 19:11:16,884] - 
  INFO root [2018-07-02 19:11:16,885] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 19:11:16,887] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 19:11:16,888] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 19:11:16,888] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 19:11:16,923] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 19:11:16,924] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 19:11:16,939] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 29092. PID: 3404 ----
  INFO root [2018-07-02 19:11:16,939] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 19:11:16,940] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 19:11:16,941] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 19:11:17,117] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 19:11:17,119] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 19:11:17,120] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 19:11:17,127] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 19:11:17,128] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 19:11:17,128] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 19:11:17,130] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 19:11:17,215] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 19:11:17,301] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 19:11:17,302] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 19:11:17,328] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 19:11:17,350] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 19:11:17,925] - ---- interfaces is running ----
 DEBUG root [2018-07-02 19:11:17,925] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 19:11:25,229] - Components::process(): Found all the components!
  INFO root [2018-07-02 19:11:53,650] - 
  INFO root [2018-07-02 19:11:53,651] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 19:11:53,653] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 19:11:53,653] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 19:11:53,654] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 19:11:53,669] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 19:11:53,670] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 19:11:53,680] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 29213. PID: 9312 ----
  INFO root [2018-07-02 19:11:53,681] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 19:11:53,682] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 19:11:53,683] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 19:11:53,897] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 19:11:53,899] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 19:11:53,900] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 19:11:53,907] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 19:11:53,908] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 19:11:53,908] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 19:11:53,909] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 19:11:53,999] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 19:11:54,093] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 19:11:54,093] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 19:11:54,119] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 19:11:54,140] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 19:11:54,707] - ---- interfaces is running ----
 DEBUG root [2018-07-02 19:11:54,707] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 19:12:01,956] - Components::process(): Found all the components!
S_INFO root [2018-07-02 19:12:29,721] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 19:12:37,821] - onRequestAccountLogin: registerName=lqt01
  WARN root [2018-07-02 19:15:52,020] - TCPPacketReceiver::processPendingEvents(127.0.0.1:2563): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 19:15:57,157] - 
  INFO root [2018-07-02 19:15:57,174] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 19:15:57,176] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 19:15:57,177] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 19:15:57,178] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 19:15:57,238] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 19:15:57,238] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 19:15:57,262] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 30009. PID: 8648 ----
  INFO root [2018-07-02 19:15:57,263] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 19:15:57,264] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 19:15:57,265] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 19:15:57,469] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 19:15:57,470] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 19:15:57,471] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 19:15:57,478] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 19:15:57,479] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 19:15:57,480] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 19:15:57,481] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 19:15:57,633] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 19:15:57,720] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 19:15:57,720] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 19:15:57,745] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 19:15:57,767] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 19:15:58,856] - ---- interfaces is running ----
 DEBUG root [2018-07-02 19:15:58,857] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 19:16:06,147] - Components::process(): Found all the components!
S_INFO root [2018-07-02 19:16:25,547] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 19:16:35,947] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 19:16:39,348] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 19:16:45,448] - onRequestAccountLogin: registerName=lqt02
  WARN root [2018-07-02 19:20:02,234] - TCPPacketReceiver::processPendingEvents(127.0.0.1:2710): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 19:21:09,179] - 
  INFO root [2018-07-02 19:21:09,184] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 19:21:09,185] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 19:21:09,186] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 19:21:09,187] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 19:21:09,205] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 19:21:09,206] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 19:21:09,217] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 31028. PID: 5144 ----
  INFO root [2018-07-02 19:21:09,220] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 19:21:09,230] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 19:21:09,231] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 19:21:09,424] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 19:21:09,425] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 19:21:09,426] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 19:21:09,433] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 19:21:09,434] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 19:21:09,434] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 19:21:09,435] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 19:21:09,519] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 19:21:09,607] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 19:21:09,607] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 19:21:09,635] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 19:21:09,659] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 19:21:10,313] - ---- interfaces is running ----
 DEBUG root [2018-07-02 19:21:10,314] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 19:21:17,494] - Components::process(): Found all the components!
S_INFO root [2018-07-02 19:21:34,538] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 19:21:42,637] - onRequestAccountLogin: registerName=lqt02
  WARN root [2018-07-02 20:16:29,731] - TCPPacketReceiver::processPendingEvents(127.0.0.1:2849): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 21:20:33,805] - 
  INFO root [2018-07-02 21:20:33,806] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 21:20:33,809] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 21:20:33,809] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 21:20:33,810] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 21:20:33,821] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 21:20:33,822] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 21:20:33,833] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 21655. PID: 9960 ----
  INFO root [2018-07-02 21:20:33,834] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 21:20:33,835] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 21:20:33,836] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 21:20:34,053] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 21:20:34,055] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 21:20:34,056] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 21:20:34,063] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 21:20:34,064] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 21:20:34,064] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 21:20:34,066] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 21:20:34,307] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 21:20:34,498] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 21:20:34,498] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 21:20:34,560] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 21:20:34,585] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 21:20:35,771] - ---- interfaces is running ----
 DEBUG root [2018-07-02 21:20:35,778] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 21:20:42,738] - 
  INFO root [2018-07-02 21:20:42,739] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 21:20:42,741] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 21:20:42,742] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 21:20:42,743] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 21:20:42,757] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 21:20:42,758] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 21:20:42,772] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 21684. PID: 8592 ----
  INFO root [2018-07-02 21:20:42,773] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 21:20:42,774] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 21:20:42,775] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 21:20:42,971] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 21:20:42,972] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 21:20:42,973] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 21:20:42,981] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 21:20:42,982] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 21:20:42,983] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 21:20:42,985] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 21:20:43,079] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 21:20:43,185] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 21:20:43,186] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 21:20:43,213] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 21:20:43,238] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 21:20:44,625] - ---- interfaces is running ----
 DEBUG root [2018-07-02 21:20:44,626] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 21:20:52,214] - Components::process(): Found all the components!
S_INFO root [2018-07-02 21:21:31,916] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 21:21:40,617] - onRequestAccountLogin: registerName=lqt02
  WARN root [2018-07-02 21:27:18,716] - TCPPacketReceiver::processPendingEvents(127.0.0.1:3872): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 21:27:26,763] - 
  INFO root [2018-07-02 21:27:26,764] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 21:27:26,767] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 21:27:26,768] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 21:27:26,768] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 21:27:26,783] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 21:27:26,784] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 21:27:26,798] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 23004. PID: 1460 ----
  INFO root [2018-07-02 21:27:26,799] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 21:27:26,799] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 21:27:26,800] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 21:27:26,989] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 21:27:26,991] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 21:27:26,992] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 21:27:27,000] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 21:27:27,002] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 21:27:27,002] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 21:27:27,004] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 21:27:27,109] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 21:27:27,221] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 21:27:27,221] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 21:27:27,246] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 21:27:27,270] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 21:27:28,269] - ---- interfaces is running ----
 DEBUG root [2018-07-02 21:27:28,270] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 21:27:35,446] - Components::process(): Found all the components!
S_INFO root [2018-07-02 21:28:16,348] - onRequestAccountLogin: registerName=lqt01
S_INFO root [2018-07-02 21:28:25,148] - onRequestAccountLogin: registerName=lqt02
  WARN root [2018-07-02 22:40:17,586] - TCPPacketReceiver::processPendingEvents(127.0.0.1:4044): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 22:48:37,671] - 
  INFO root [2018-07-02 22:48:37,677] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 22:48:37,682] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 22:48:37,684] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 22:48:37,684] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 22:48:37,719] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 22:48:37,720] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 22:48:37,736] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 6142. PID: 3676 ----
  INFO root [2018-07-02 22:48:37,739] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 22:48:37,742] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 22:48:37,744] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 22:48:37,964] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 22:48:37,966] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 22:48:37,967] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 22:48:37,978] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 22:48:37,979] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 22:48:37,980] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 22:48:37,982] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 22:48:38,104] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 22:48:38,227] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 22:48:38,227] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 22:48:38,258] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-07-02 22:48:38,286] - TelnetServer server is running on port 33000
  INFO root [2018-07-02 22:48:39,302] - ---- interfaces is running ----
 DEBUG root [2018-07-02 22:48:39,302] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 22:48:46,560] - Components::process(): Found all the components!
S_INFO root [2018-07-02 23:06:35,191] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 23:06:48,391] - onRequestAccountLogin: registerName=lqt01
  INFO root [2018-07-02 23:08:05,098] - 
  INFO root [2018-07-02 23:08:05,099] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 23:08:05,101] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 23:08:05,101] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 23:08:05,101] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 23:08:05,111] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 23:08:05,111] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 23:08:05,118] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 9956. PID: 8256 ----
  INFO root [2018-07-02 23:08:05,119] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 23:08:05,119] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 23:08:05,119] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 23:08:05,361] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 23:08:05,362] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 23:08:05,363] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 23:08:05,380] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 23:08:05,381] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 23:08:05,381] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 23:08:05,383] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 23:08:05,507] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 23:08:05,632] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 23:08:05,632] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 23:08:05,653] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
 S_ERR root [2018-07-02 23:08:05,713] - Traceback (most recent call last):
  File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\kbemain.py", line 45, in onInterfaceAppReady
    g_poller.start("localhost", 30040)
 S_ERR root [2018-07-02 23:08:05,726] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\Poller.py", line 30, in start
    self._socket.bind((addr, port))
 S_ERR root [2018-07-02 23:08:05,728] - OSError: [WinError 10048] 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
  INFO root [2018-07-02 23:08:05,739] - TelnetServer server is running on port 33001
  INFO root [2018-07-02 23:08:06,386] - ---- interfaces is running ----
 DEBUG root [2018-07-02 23:08:06,387] - Components::process(): Request for the process of identity...
 ERROR root [2018-07-02 23:08:06,512] - Components::process: found interfaces, addr:192.168.31.56:30099
 ERROR root [2018-07-02 23:08:06,560] - ServerApp::onIdentityillegal: The current process and interfaces(componentID=3000 ->conflicted???, pid=3676, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [2018-07-02 23:08:06,592] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [2018-07-02 23:08:06,593] - onInterfaceAppShutDown()
  INFO root [2018-07-02 23:08:07,630] - Shutdowner::onShutdown: shutting down
  INFO root [2018-07-02 23:08:08,639] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-07-02 23:08:08,660] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 23:08:08,660] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 23:08:08,661] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 23:08:08,661] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 23:08:08,661] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 23:08:08,662] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 23:08:08,662] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 23:08:08,662] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 23:08:08,691] - Script::uninstall(): is successfully!
 DEBUG root [2018-07-02 23:08:08,701] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-07-02 23:08:09,002] - ThreadPool::destroy(): waiting for thread(2)[0x6a43a8,0x6a4340,], try=1
 DEBUG root [2018-07-02 23:08:09,402] - TPThread::~TPThread(): 0x6a43a8
 DEBUG root [2018-07-02 23:08:09,402] - TPThread::~TPThread(): 0x6a4340
 DEBUG root [2018-07-02 23:08:09,402] - ThreadPool::destroy(): successfully!
 DEBUG root [2018-07-02 23:08:09,429] - Bundle::destroyObjPool(): size 1.
 DEBUG root [2018-07-02 23:08:09,429] - Channel::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:08:09,429] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:08:09,430] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2018-07-02 23:08:09,430] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:08:09,440] - Address::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:08:09,440] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:08:09,440] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2018-07-02 23:08:09,440] - interfaces(3000) has shut down.
  INFO root [2018-07-02 23:08:10,440] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2018-07-02 23:10:25,042] - 
  INFO root [2018-07-02 23:10:25,043] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 23:10:25,045] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 23:10:25,046] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 23:10:25,047] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 23:10:25,083] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 23:10:25,084] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 23:10:25,096] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 10410. PID: 8880 ----
  INFO root [2018-07-02 23:10:25,097] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 23:10:25,097] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 23:10:25,098] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 23:10:25,287] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 23:10:25,288] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 23:10:25,290] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 23:10:25,297] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 23:10:25,298] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 23:10:25,299] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 23:10:25,300] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 23:10:25,400] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 23:10:25,500] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 23:10:25,500] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 23:10:25,530] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
 S_ERR root [2018-07-02 23:10:25,555] - Traceback (most recent call last):
  File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\kbemain.py", line 45, in onInterfaceAppReady
    g_poller.start("localhost", 30040)
 S_ERR root [2018-07-02 23:10:25,558] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\Poller.py", line 30, in start
    self._socket.bind((addr, port))
 S_ERR root [2018-07-02 23:10:25,560] - OSError: [WinError 10048] 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
  INFO root [2018-07-02 23:10:25,567] - TelnetServer server is running on port 33001
  INFO root [2018-07-02 23:10:26,641] - ---- interfaces is running ----
 DEBUG root [2018-07-02 23:10:26,643] - Components::process(): Request for the process of identity...
 ERROR root [2018-07-02 23:10:26,916] - Components::process: found interfaces, addr:192.168.31.56:30099
 ERROR root [2018-07-02 23:10:26,917] - ServerApp::onIdentityillegal: The current process and interfaces(componentID=3000 ->conflicted???, pid=3676, addr=192.168.31.56) conflict, the process will exit!
Can modify the components-CID and UID to avoid conflict.
  INFO root [2018-07-02 23:10:26,919] - Shutdowner::onShutdownBegin: shutting down(period=1, tickPeriod=1)
S_INFO root [2018-07-02 23:10:26,920] - onInterfaceAppShutDown()
  INFO root [2018-07-02 23:10:27,921] - Shutdowner::onShutdown: shutting down
  INFO root [2018-07-02 23:10:28,922] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2018-07-02 23:10:28,923] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 23:10:28,923] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 23:10:28,924] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 23:10:28,924] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 23:10:28,925] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 23:10:28,925] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 23:10:28,925] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 23:10:28,926] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 23:10:28,947] - Script::uninstall(): is successfully!
 DEBUG root [2018-07-02 23:10:28,948] - ThreadPool::destroy(): starting size 2.
  WARN root [2018-07-02 23:10:29,253] - ThreadPool::destroy(): waiting for thread(2)[0x4543c0,0x454358,], try=1
 DEBUG root [2018-07-02 23:10:29,654] - TPThread::~TPThread(): 0x4543c0
 DEBUG root [2018-07-02 23:10:29,654] - TPThread::~TPThread(): 0x454358
 DEBUG root [2018-07-02 23:10:29,654] - ThreadPool::destroy(): successfully!
 DEBUG root [2018-07-02 23:10:29,670] - Bundle::destroyObjPool(): size 1.
 DEBUG root [2018-07-02 23:10:29,670] - Channel::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:10:29,670] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:10:29,670] - UDPPacket::destroyObjPool(): size 14.
 DEBUG root [2018-07-02 23:10:29,671] - EndPoint::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:10:29,671] - Address::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:10:29,671] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2018-07-02 23:10:29,671] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2018-07-02 23:10:29,671] - interfaces(3000) has shut down.
  INFO root [2018-07-02 23:10:30,671] - EventDispatcher()::~EventDispatcher: Num timers = 2
  WARN root [2018-07-02 23:13:13,744] - TCPPacketReceiver::processPendingEvents(127.0.0.1:4469): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2018-07-02 23:13:39,915] - 
  INFO root [2018-07-02 23:13:39,915] - -----------------------------------------------------------------------------------------


  INFO root [2018-07-02 23:13:39,917] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [2018-07-02 23:13:39,917] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [2018-07-02 23:13:39,917] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [2018-07-02 23:13:39,927] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:30099 )
  INFO root [2018-07-02 23:13:39,928] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:30099, SOMAXCONN=5.
  INFO root [2018-07-02 23:13:39,936] - ---- interfaces Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: 1ABB6125D8365C3720079E1D4CCC25C7. Config: Debug X86. Built: 23:15:38 Jun 15 2018. AppID: 3000. UID: 11050. PID: 7564 ----
  INFO root [2018-07-02 23:13:39,936] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [2018-07-02 23:13:39,937] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-07-02 23:13:39,938] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-07-02 23:13:40,214] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-07-02 23:13:40,215] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-07-02 23:13:40,216] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-07-02 23:13:40,224] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-07-02 23:13:40,224] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-07-02 23:13:40,225] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-07-02 23:13:40,226] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-07-02 23:13:40,346] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-07-02 23:13:40,475] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:15:28) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-07-02 23:13:40,476] - Script::install(): KBExtra is successfully!
S_INFO root [2018-07-02 23:13:40,536] - onInterfaceAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
 S_ERR root [2018-07-02 23:13:40,627] - Traceback (most recent call last):
  File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\kbemain.py", line 45, in onInterfaceAppReady
    g_poller.start("localhost", 30040)
 S_ERR root [2018-07-02 23:13:40,629] -   File "C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/interface\Poller.py", line 30, in start
    self._socket.bind((addr, port))
 S_ERR root [2018-07-02 23:13:40,630] - OSError: [WinError 10048] 通常每个套接字地址(协议/网络地址/端口)只允许使用一次。
  INFO root [2018-07-02 23:13:40,639] - TelnetServer server is running on port 33001
  WARN root [2018-07-02 23:13:40,639] - c:\users\public\documents\workspace\kbe\kbengine-kbengine-v1.0.0\kbengine\kbe\src\server\tools\interfaces\interfaces.cpp:164: Profile scriptCall took 0.10 seconds
  INFO root [2018-07-02 23:13:41,457] - ---- interfaces is running ----
 DEBUG root [2018-07-02 23:13:41,458] - Components::process(): Request for the process of identity...
  INFO root [2018-07-02 23:13:48,737] - Components::process(): Found all the components!
S_INFO root [2018-07-02 23:14:13,605] - onRequestAccountLogin: registerName=lqt02
S_INFO root [2018-07-02 23:14:21,505] - onRequestAccountLogin: registerName=lqt01
  WARN root [2018-07-02 23:15:43,720] - TCPPacketReceiver::processPendingEvents(127.0.0.1:5246): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
